### PR TITLE
Bekker lecture

### PR DESCRIPTION
**Pull request by How2CodeGood**
### What

Adding Lecture to website, put it in the wrong repository but changed that, and now it should work fine

### Why

Lecture Tomorrow

### Where

In student-lectures folder

### Testing

Tested that the website works well in my forked repository 

### Additional Notes

Please let me know if I need a bibliography of some sort

### CODE REVIEW

File: site/student-lectures/Evolutionary_Art/electric_sheep_example.gif
Comment (jameshughes89) [inline site/student-lectures/Evolutionary_Art/electric_sheep_example.gif:1]: This file is rather large. Can you make it smaller

**Review by jameshughes89 — CHANGES_REQUESTED** [review]
1. You need to add your lecture to the index.rst file. Just look at the current index.rst and see how others did it. 
2. Can you confirm that you built the website and checked that it worked correctly?
3. It feels like it's missing _something_. Perhaps add more content.

File: site/student-lectures/Evolutionary_Art/electric_sheep_example.gif
Comment (How2CodeGood) [reply site/student-lectures/Evolutionary_Art/electric_sheep_example.gif:1]: Done, 1.77Mb now

**Comment by How2CodeGood** [comment]
What's the input for source directory & output directory for shinx build?

**Comment by jameshughes89** [comment]
> What's the input for source directory & output directory for shinx build?

Source is `site` and output is whatever you want, but probably make a dir called `build` and make it that.

**Comment by jameshughes89** [comment]
> Now that I have the build dir, what do I do with it? Can I test the website past the preview option I get on the github dir?

I am not sure I understand your question. 

building the website with sphinx put the built website into the directory you specify (presumably the build directory). Open that directory and open the index.html file and/or open your lecture's html file.

**Comment by How2CodeGood** [comment]
> 1. You need to add your lecture to the index.rst file. Just look at the current index.rst and see how others did it.
> 
>     2. Can you confirm that you built the website and checked that it worked correctly?
> 
>     3. It feels like it's missing _something_. Perhaps add more content.

Much of the presentation is going to be spent exploring different evolutionary art models, going to be practicing and timing a lot today, might add another secotion after the variations section.

**Comment by How2CodeGood** [comment]
> > Now that I have the build dir, what do I do with it? Can I test the website past the preview option I get on the github dir?
> 
> I am not sure I understand your question.
> 
> building the website with sphinx put the built website into the directory you specify (presumably the build directory). Open that directory and open the index.html file and/or open your lecture's html file.

I understand now, it ran perfectly, should be fine to upload

**Review by jameshughes89 — APPROVED** [review]

**Pull request merged**